rustls/src/quic.rs

```diff
@@ -771,6 +772,7 @@ pub trait PacketKey: Send + Sync {
 }
 
 /// Packet protection keys for bidirectional 1-RTT communication
+#[non_exhaustive]
```

Comment:
And both this and `Keys` feel like they could probably stick to being exhaustive, though I didn't find any Quinn code relying on this.